moment-element.html

```diff
@@ -33,80 +33,77 @@
          * For consistent results, parsing anything other than ISO 8601 strings
          * with the `inputFormat` property. More information in [moment String](http://momentjs.com/docs/#/parsing/string/).
          */
-        datetime: String,
+        datetime: {
+          type: String,
+          value: new Date().getTime()
```

Comment:
Please, use a function to ensure that each element gets its own copy of the value. And better use simple `new Date()`:

```javascript
value: function() {
  return new Date();
}
```